### PR TITLE
Fix IE support

### PR DESCRIPTION
**Pull request by jalkoby**
The fix for #386

### CODE REVIEW

File: src/utils/setImmediate.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
-const setImmediate = typeof global !== 'undefined' && typeof global.setImmediate !== 'undefined' ? global.setImmediate : (fn, ms) => setTimeout(fn, ms)
+const hasNativeSupport = typeof global !== 'undefined' && typeof global.setImmediate !== 'undefined'
+const setImmediate = hasNativeSupport ? (fn, ms) => global.setImmediate : (fn, ms) => setTimeout(fn, ms)
```
Comment (Andarist) [inline src/utils/setImmediate.js]: returned functions returns `global.setImmediate` instead of calling it with provided arguments ;)

**Comment by jalkoby** [comment]
Sorry - rush fixing (without running test suite at the end) is not a good idea

**Comment by Andarist** [comment]
Thanks! Cannot publish it though yet, needs to be done by @rt2zz

**Comment by Andarist** [comment]
@jalkoby just released with a fix, please check it out :)

**Comment by jalkoby** [comment]
thanks

**Pull request merged**